app/models/piece.rb

```diff
@@ -1,3 +1,81 @@
 class Piece < ActiveRecord::Base
   belongs_to :game
+
+  def check_square(x, y)
+    return true if @game.pieces.where(row_coordinate: x, column_coordinate: y).any?
```

Comment:
In Ruby, methods always return the last value in a method.

So here, the `return` isn't necessary. You could do this.

```Ruby
def check_square(row, column)
  game.pieces.where(row_coordinate: x, column_coordinate: y).any?
end